src/interfaces/IERC4973Permit.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.6;
+
+/// @title Account-bound tokens, optional permissioned minting extension
+/// @dev See https://eips.ethereum.org/EIPS/eip-4973
```

Comment:
please add the function identifier as a hex value here as in the other interface: https://github.com/rugpullindex/ERC4973/pull/18/files#diff-d6d069dfcb878b5b981b7d3dd18dce17746639eea3f30d1902f4a4cd46364f84R6

You can generate the interface identifier with a similar test as this one: https://github.com/rugpullindex/ERC4973/blob/9c6bd57c4f34da5a1507ec2ff6cb3b105594d712/src/ERC4973.t.sol#L50